meetings/working-groups/collection-literals/CL-LDM-05-31-2023.md

```diff
@@ -1,10 +1,74 @@
-# C# LDM Collection Literals Agenda
+[# C# LDM Collection Literals Agenda
 
 ## Core set of features for C# 12
 
 ### Target-typing to core collections and collections with classic collection initializers
 
-https://github.com/dotnet/csharplang/blob/main/proposals/collection-literals.md#constructible-collection-types-target-types
+The following are constructible collection types that can be used to construct collection literals. These are the valid target types for collection literals.
+
+The list is in priority order. If a collection type belongs in multiple categories, the first is used.
+
+
+1. Single dimensional arrays (e.g. T[])
+
+```c#
+int[] a = [1, 2, 3]; // equivalent to:
+int[] a = new int[] { 1, 2, 3 };
+```
+
+2. Span types (`Span<T>` and `ReadOnlySpan<T>`)
+
+```c#
+Span<int> a = [1, 2, 3]; // equivalent to either:
+
+Span<int> a = new int[] { 1, 2, 3 }; // or
+Span<int> a = stackalloc int[] { 1, 2, 3 };
+```
+
+Which one is picked depends on compiler heuristics around stack space (and is part of the 'params span' WG outcomes), and the ref-safety rules around 'a'.
+
+3. Types with a suitable Construct method.  This will be discussed below, but relates heavily to immutable collections, and how to efficiently construct them.
```

Comment:
```suggestion
3. Types with a suitable Construct method.  This will be discussed below, but relates heavily to immutable collections, and how to efficiently construct them.  It will also be the preferred way to create a `List<T>` as it is the most efficient form with lowest overhead.
```